README.md

```diff
@@ -6,6 +6,7 @@ Kibana is an open source ([Apache Licensed](https://github.com/elastic/kibana/bl
 
 - Elasticsearch master
 - Kibana binary package
+- 512 MB of available RAM
```

Comment:
512 here? I assume this is because some amount of memory will of course be used that isn't old space